src/wallet/isEntrypoint.ts

```diff
@@ -0,0 +1,10 @@
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+import { isEntrypointOptions, isEntrypointResponse } from './types';
+
+export function isEntrypoint({ entrypoint }: isEntrypointOptions): isEntrypointResponse {
```

Comment:
I am not sure this is the right wording for this function. `is` `Entrypoint` is way too generic.

What are we actually trying to do here?

Reply:
What is Entrypoint anyway?

Reply:
We are trying to validate that the specified entrypoint contract is correct. It should be "version 6"

Reply:
`isEntrypointContractV6`?

Reply:
Entrypoint is the address for something, and back looking to `willSponsor` where the `entrypoint` is coming from?

Is that coming from Paymaster service? or from where?

Reply:
The entrypoint is specified on the client side when they initialize the `paymasterClient`